mods/ts/rules/shared-support.yaml

```diff
@@ -9,6 +9,7 @@ NAPULS:
 		Queue: Defense
 		BuildPaletteOrder: 90
 		Prerequisites: radar
+		BuildLimit: 1
```

Comment:
I'm not sure if this is like that in TS. Did u added that because it doesn't work yet.

Original ones only gives range support doesn't give new sw.